src/c++/perf_analyzer/main.cc

```diff
@@ -26,9 +27,13 @@
 
 #include <getopt.h>
 #include <signal.h>
+
```

Comment:
Why add gaps here?

Reply:
It's just what my `clang-format` gives me when following the `.clang-format` file in the repo root.

My `clang-format` is version is `clang-format version 10.0.0-4ubuntu1`. This is what gets installed automatically by `apt update ; apt install -y clang-format` in ubuntu 20.04.2.

I'm using the build docker container (whose base OS is ubuntu 20.04.2) to develop in since it has all of the build dependencies.

I see the `format.py` file you linked expects `clang-format` version 6.0. Do you know how I can install `clang-format` version 6.0 in ubuntu 20.04.2?

Reply:
> @jbkyang-nvi
> 
> > Are you still working on the changes to pipe these variables through?
> 
> Yes, I was planning on putting that in a separate PR. Would you prefer those changes to be in this PR?

I would prefer if you:
1. Create a merge queue where both PRs can can be merged together (they can be reviewed separately)
2. Add to each of the option added "WIP feature, does not work". And removed them in the second PR